### PR TITLE
typo

### PR DESCRIPTION
**Pull request by nitedog**
corrected link text to "Focus Appearance" by removing the word "(Minimum)"

### CODE REVIEW

**Comment by rscano** [comment]
Shall also the URL of the tech updated, removing "minimum" word?

File: understanding/20/focus-visible.html [hunk 1]
```diff
@@ -23,7 +23,7 @@ <h2>Intent of Focus Visible</h2>
          <p>There may be situations where mouse/pointer users could also benefit from having a visible focus indicator, even though they did not set focus to an element using the keyboard. As a best practice, consider still providing an explicit focus indicator for these cases.</p>
       </div>
       
-   	<p>Note that a keyboard focus indicator can take different forms.<span class="wcag22"> This criterion does not specify what the form is, but <a href="focus-appearance-minimum">Focus Appearance</a> does define how visible the indicator should be. Passing <a href="focus-appearance-minimum">Focus Appearance (Minimum)</a> would pass this success criterion.</span></p>
```
Comment (fstrr) [inline understanding/20/focus-visible.html:26]: You also need to remove the `-minimum` from the URL.

File: understanding/20/focus-visible.html [hunk 2]
```diff
@@ -23,7 +23,7 @@ <h2>Intent of Focus Visible</h2>
...
+   	<p>Note that a keyboard focus indicator can take different forms.<span class="wcag22"> This criterion does not specify what the form is, but <a href="focus-appearance-minimum">Focus Appearance</a> does define how visible the indicator should be. Passing <a href="focus-appearance-minimum">Focus Appearance</a> would pass this success criterion.</span></p>
```
Comment (patrickhlauke) [inline understanding/20/focus-visible.html:26]:
```suggestion
   	<p>Note that a keyboard focus indicator can take different forms.<span class="wcag22"> This criterion does not specify what the form is, but <a href="focus-appearance">Focus Appearance</a> does define how visible the indicator should be. Passing <a href="focus-appearance">Focus Appearance</a> would pass this success criterion.</span></p>
```

**Comment by patrickhlauke** [comment]
this PR seems redundant as #3272 has already been corrected. it now creates a merge conflict because we worded things differently

**Comment by nitedog** [comment]
apologies for causing confusion -- would closing this PR help resolve the conflict?

**Comment by patrickhlauke** [comment]
No need to apologise...we're all monkey-patching around (half the time, my PRs even from a few weeks ago start drifting out of sync with whatever else gets merged in the meantime). But yes, I'll close this PR if you don't mind.

**Pull request closed without merging**